### PR TITLE
Add more information to Kubernetes entities

### PR DESCRIPTION
**Pull request by aermakov-zalando**
* All Kubernetes entities: add `namespace`, `name`, `deletion_timestamp`
 * Containers: add `start_time`, `running`

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
@arjunrn @vetinari PTAL

File: zmon_agent/discovery/kubernetes/cluster.py [hunk 1]
```diff
@@ -334,14 +345,27 @@ def get_cluster_pods_and_containers(
...
 
+            start_time = None
+            container_running = False
+
+            container_state = container_status.get("state", {})
+            if "running" in container_state:
+                start_time = container_state["running"]["startedAt"]
+                container_running = True
+            elif "terminated" in container_state:
+                start_time = container_state["running"]["startedAt"]
```
Comment (arjunrn) [inline zmon_agent/discovery/kubernetes/cluster.py]: should be `container_state["terminated"]["startedAt"]`

Comment (aermakov-zalando) [reply zmon_agent/discovery/kubernetes/cluster.py]: Thanks! Fixed.

**Comment by vetinari** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**